src/components/FileDropzone.js

```diff
@@ -27,6 +27,7 @@ function FileDropzone({ setPDFFile }) {
       setPDFFile(null);
       setDropzoneText(dropzoneError);
     }
+    setPDFFile(selectedFile);
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why eslint disable?

Reply:
I do not know, in fact, I haven't disabled it myself.